project/Publish.scala

```diff
@@ -61,27 +63,3 @@ object NoPublish extends AutoPlugin {
   override def projectSettings =
     Seq(skip in publish := true, sources in (Compile, doc) := Seq.empty, whitesourceIgnore := true)
 }
-
-object DeployRsync extends AutoPlugin {
-  import scala.sys.process._
-  import sbt.complete.DefaultParsers._
-
-  override def requires = plugins.JvmPlugin
-
-  trait Keys {
-    val deployRsyncArtifacts = taskKey[Seq[(File, String)]]("File or directory and a path to deploy to")
-    val deployRsync = inputKey[Unit]("Deploy using rsync")
-  }
-
-  object autoImport extends Keys
-  import autoImport._
-
-  override def projectSettings = Seq(
-    deployRsyncArtifacts := List(),
-    deployRsync := {
-      val (_, host) = (Space ~ StringBasic).parsed
-      deployRsyncArtifacts.value.foreach {
-        case (from, to) => s"rsync -rvz $from/ $host:$to" !
```

Comment:
The new is:
```
Process(Seq("rsync", "-azP", s"$from/", s"$toHost:$toPath"), None, "RSYNC_RSH" -> "ssh -o StrictHostKeyChecking=no")
```

Are the differences ok?

Reply:
The changes are:

* remove `-r`: already implied by `-a`
* remove `-v`: verbose, seems fine to skip
* add `-a`: seems like an improvement (recursive, keep symlinks, permissions and timestamps)
* add '-P': show progress. not really necessary but OK, could consider dropping upstream
* RSYNC_RSH: not sure, but seems OK
* `StrictHostKeyChecking=no` not sure if needed, could consider dropping upstream

So in short: I don't see anything harmful upstream.

Reply:
thanks for clarifying

Reply:
At least on mac `-a` essentially is an alias of `-rlptgoD` - most of those seem ok, but what about `-g` and `-o` which is preserve owner and group. I'm also curious about `-D` which "is the same as --devices and --specials" which I don't find any further details about.

Reply:
Now I found `--specials`

> This option causes rsync to transfer special files such as named sockets and fifos.

Ehm?

Reply:
I think the group/owner things only apply when you're root, so they don't apply for 'regular use'. I might be wrong, but I didn't notice any problems when using this to publish to gustav

Indeed it wouldn't make sense to rsync over special/device files, but they wouldn't be generated in our docs, so I'm not sure if it matters?

I'd be fine with dropping some of the flags over at sbt-publish-rsync, but perhaps it doesn't help anything?

Reply:
If it doesn't do anything weird with owner and group I think it's ok